src/content/docs/apm/agents/go-agent/instrumentation/distributed-tracing-go-agent.mdx

```diff
@@ -0,0 +1,439 @@
+    2. From the trace details page, click **See logs**.
+    3. To view details related to an individual log message, click directly on the message.
+  </Collapser>
+</CollapserGroup>
+
+For more help finding your traces in the UI:
+
+* [Understand and use the distributed tracing UI](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui)
+* [Query distributed trace data](/docs/understand-dependencies/distributed-tracing/ui-data/query-distributed-trace-data)
+
+## Set up Infinite Tracing (advanced option) [#infinite-tracing]
+
+Standard distributed tracing for APM agents (above) use [adaptive sampling](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#trace-origin-sampling) to capture up to 10 traces per minute, but if you want us to analyze all your data and find the most relevant traces, you can set up Infinite Tracing.
+
+<Callout variant="tip">
+  To learn more about this feature, see [Infinite Tracing](/docs/understand-dependencies/distributed-tracing/infinite-tracing/introduction-infinite-tracing).
+</Callout>
+
+Before beginning, first ensure you meet [the requirements](/docs/understand-dependencies/distributed-tracing/infinite-tracing/introduction-infinite-tracing#requirements).
+
+### Step 1. Complete the new agent installation [#install-agents-first] 
+
+The Infinite Tracing setup builds on the instrumentation step from the [New agent installations](#quick-start-apm) for standard distributed tracing.
+
+### Step 2. Set up the trace observer [#provision-trace-observer]
+
+The trace observer is a New Relic AWS-based service that collects and analyzes all your traces. Follow the instructions in [Set up trace observer](/docs/understand-dependencies/distributed-tracing/infinite-tracing/set-trace-observer). When you're done, return here with your trace observer information and continue with the next step to configure the agent.
+
+### Step 3: Configure the agent for Infinite Tracing
+
+Infinite Tracing configuration settings include the standard distributed tracing plus information about the trace observer:
+
+<CollapserGroup>
+  <Collapser
+    id="go-config"
+    title="Go Infinite Tracing configuration"
+  >
+    Here's an overview of the settings.
+
+    <table>
+      <thead>
+        <tr>
+          <th style={{ width: "200px" }}>
+            Type
+          </th>
+
+          <th>
+            Required configuration
+          </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>
+            Infinite Tracing
+          </td>
+
+          <td>
+            Configuration options:
+
+            * `newrelic.Config` structure:
+
+              ```
+              app, err := newrelic.NewApplication(
+                  newrelic.ConfigAppName(<var>YOUR_APP_NAME</var>),
+                  newrelic.ConfigLicense(<var>YOUR_LICENSE_KEY</var>),
+                  func(cfg *newrelic.Config) {
+                      cfg.DistributedTracer.Enabled = true
```

Comment:
We shouldn't need `cfg.DistributedTracer.Enabled = true` since DT is on by default.
Also, I think it would be better to use `newrelic.ConfigFromEnvironment()` which will set a plethora of configuration options from environment variables at runtime. In particular this means their New Relic license key can be loaded form the environment and not hardcoded in the source code where it will likely be stored in their code repository.
At least it's worth a mention in the documentation.

Reply:
This recommendation is actually for all examples in our documentation where we show configuration of the agent.